kedro-airflow/pyproject.toml

```
@@ -13,7 +13,7 @@ license = {text = "Apache Software License (Apache 2.0)"}
 dependencies = [
     "kedro>=0.17.5",
     "python-slugify>=4.0",
-    "semver~=2.10",  # Needs to be at least 2.10.0 to get VersionInfo.match
+    "semver~=3.0",
```

```suggestion
    "semver>=2.10",  # Needs to be at least 2.10.0 to make use of `VersionInfo.match`.
```
Can we just relax it instead? Feels unnecessary to drop 2.x compatibility, for anybody who may need that.

---

Exact text copied from https://github.com/kedro-org/kedro-viz/pull/1452/files